cornichon-docs/src/main/tut/dsl.md

```diff
@@ -274,6 +274,85 @@ And assert body.ignoring("city", "publisher").is(
 ```
 
 
+## Kafka Support (experimental)
```

Comment:
I would prefer having this in a different file as it is not shipped by default.
You could create `kafka.md` and add it last [here](https://github.com/agourlay/cornichon/blob/master/cornichon-docs/src/main/resources/microsite/data/menu.yml) so it gets its own menu.

Reply:
done